com.unity.netcode.gameobjects/Tests/Runtime/MultiInstanceHelpers.cs

```diff
@@ -155,9 +157,21 @@ public static bool Start(bool host, NetworkManager server, NetworkManager[] clie
                 server.StartServer();
             }
 
+            // if set, then invoke this for the server
+            if (startInitializationCallback != null)
+            {
+                startInitializationCallback.Invoke(server);
```

Comment:
you can do startInitializationCallback?.Invoke(server); with the question mark, without the null check around

Reply:
Yep!  Done!